src/bootstrap/Cargo.lock

```diff
@@ -88,9 +88,8 @@ dependencies = [
 
 [[package]]
 name = "cc"
-version = "1.1.22"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "9540e661f81799159abee814118cc139a2004b3a3aa3ea37724a1b66530b90e0"
+version = "1.2.0"
+source = "git+https://github.com/nhtyy/cc-rs?branch=n%2Fadd-succinct-target#424acb084e212c330de96e06cab69be0d9df55f4"
```

Comment:
Problem: this is also pointing to a custom `cc`-rs fork that I don't think we can land upstream here in rust-lang/rust.

Reply:
(Same review comment chain as the bootstrap `cc` version).